tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -205,7 +205,7 @@ stages:
                   --build_wheel \
                   --enable_onnx_tests --use_cuda --cuda_version=11.1 --cuda_home=/usr/local/cuda-11.1 --cudnn_home=/usr/local/cuda-11.1 \
                   ${{ parameters.build_py_parameters }} \
-                  --cmake_extra_defines CMAKE_CUDA_HOST_COMPILER=/opt/rh/devtoolset-9/root/usr/bin/cc 'CMAKE_CUDA_ARCHITECTURES=35;37;50;52;60;61;70;75;80;86'
+                  --cmake_extra_defines CMAKE_CUDA_HOST_COMPILER=/opt/rh/devtoolset-9/root/usr/bin/cc 'CMAKE_CUDA_ARCHITECTURES=50;52;60;61;70;75;80;86'
```

Comment:
Is this change just limited to the Python package ? As far as I can tell, the shared libraries are still with support for these cuda architectures. If so, does it cause some inconsistency ? 

Also, should we document this support pull out for these cuda architectures somewhere ? It seems like this means lack of support for K80s and K40s with the Python package (unless someone builds the wheel from source).

Reply:
Yes. I only made the change for python. Only python packages currently hit the limit.  Do you prefer consistency over functionality on the other platforms? 

> we document this support pull out for these cuda architectures somewhere 

Good suggestion! But it will be another PR to the gh-pages branch.

Reply:
Pytorch has 'sm_37', 'sm_50', 'sm_60', 'sm_61', 'sm_70', 'sm_75', 'sm_80'. Guenther  suggests we do the same. What do you think?

Reply:
Prefer if 37 is included as K80 has compute capability 3.7 and atleast one internal team uses that although they are not interested in Python. But I guess it is not the end of the world if it gets pulled out as well as we can always build from source.

Reply:
let's include 37 then.